js/chat_app/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,84 @@
+import AppBar from '@mui/material/AppBar';
+import Button from '@mui/material/Button';
+import { deepOrange } from '@mui/material/colors';
+import { useContext, useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import authSelectors from '../../redux/auth/selector';
+import Container from '@mui/material/Container';
+import Toolbar from '@mui/material/Toolbar';
+import Avatar from '@mui/material/Avatar';
+import Typography from '@mui/material/Typography';
+import Dialog from '@mui/material/Dialog';
+import DialogContent from '@mui/material/DialogContent';
+import DialogActions from '@mui/material/DialogActions';
+import Stack from '@mui/material/Stack';
+import { StyledBadge } from '../StyleBadge';
+import { ThemeProvider } from '@emotion/react';
+import { isLogout } from '../../redux/auth/slice';
+import { ThemeContext } from '../ThemeContext/ThemeContext';
+
+function Header() {
+  const [isWindowLogoutOpen, setIsWindowLogoutOpen] = useState(false);
+  const name = useSelector(authSelectors.getUsername);
+  const dispatch = useDispatch();
+  const navigation = useNavigate();
+
+  const theme = useContext(ThemeContext);
+
+  useEffect(() => {
+    navigation('/chat');
+    localStorage.getItem('auth');
```

Comment:
why do you call it here?

Reply:
if we talk about localStorage.getItem('auth') I forgot to delete it.